### PR TITLE
Add elwaziImportUrl to UI Config (#4879)

### PR DESCRIPTION
**Pull request by amarjandu**
**Description**
Part of #4879, from instructions found in https://github.com/dockstore/dockstore-ui2/blob/e229c2dcdf4e1f61fa2e3e98997bb1ef60dba160/src/app/workflow/launch-third-party/launch-third-party.component.ts#L46

**Issue**
#4879 

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [X] Check that you pass the basic style checks and unit tests by running `mvn clean install`
- [X] Follow the existing JPA patterns for queries, using named parameters, to avoid SQL injection
- [X] If you are changing dependencies, check the Snyk status check or the dashboard to ensure you are not introducing new high/critical vulnerabilities
- [X] Assume that inputs to the API can be malicious, and sanitize and/or check for Denial of Service type values, e.g., massive sizes
- [X] Do not serve user-uploaded binary images through the Dockstore API
- [X] Ensure that endpoints that only allow privileged access enforce that with the `@RolesAllowed` annotation
- [X] Do not create cookies, although this may change in the future

### CODE REVIEW

File: dockstore-integration-testing/src/test/resources/dockstore.yml [hunk 1]
```diff
@@ -106,6 +106,7 @@ uiConfig:
   dnaStackImportUrl: https://app.dnastack.com/#/app/workflow/import/dockstore
   dnaNexusImportUrl: https://platform.dnanexus.com/panx/tools/import-workflow
   terraImportUrl: https://app.terra.bio/#import-tool/dockstore
+  elwaziImportUrl: https://elwazi.terra.bio/#import-tool/dockstore
```
Comment (amarjandu) [inline dockstore-integration-testing/src/test/resources/dockstore.yml:109]: @coverbeck not sure if this is the correct URL, it's composed similarly to the `terraImportUrl` both return a `page not found` if attempting to follow the link.

Also are there configuration env files that I need to update for the various deployments? I was attempting to follow https://github.com/dockstore/dockstore-ui2/blob/e229c2dcdf4e1f61fa2e3e98997bb1ef60dba160/src/app/workflow/launch-third-party/launch-third-party.component.ts#L46

**Comment by amarjandu** [comment]
> **Note**: BitBucket tests are flaky... see thread https://ucsc-gi.slack.com/archives/C16ET3CF4/p1655845986437399

File: dockstore-integration-testing/src/test/resources/dockstore.yml (hunk 1 above)
Comment (coverbeck) [reply dockstore-integration-testing/src/test/resources/dockstore.yml:109]: Yes, that url is correct

**Review by coverbeck — COMMENTED** [review]
Looks good, but because this a hotfix, this needs to target a hotfix branch.

File: dockstore-integration-testing/src/test/resources/dockstore.yml (hunk 1 above)
Comment (coverbeck) [reply dockstore-integration-testing/src/test/resources/dockstore.yml:109]: You'll also need to update dockstore-deploy, https://github.com/dockstore/dockstore-deploy/blob/94bd66df13a433c4c1bcad5ec2b681160e475fa2/dockstore.yml.template#L160

**Review by coverbeck — APPROVED** [review]

**Review by svonworl — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]

**Pull request merged**